roles/kubernetes-apps/metallb/defaults/main.yml

```diff
@@ -6,3 +6,7 @@ metallb_port: "7472"
 metallb_limits_cpu: "100m"
 metallb_limits_mem: "100Mi"
 metallb_peers: []
+metallb_speaker_nodeselector: {}
+metallb_controller_nodeselector: {}
+metallb_speaker_tolerations: []
```

Comment:
Can we set
```
      - effect: NoSchedule
        key: node-role.kubernetes.io/master
      - effect: NoSchedule
        key: node-role.kubernetes.io/control-plane
```
as the default value of metallb_speaker_tolerations?
According to https://raw.githubusercontent.com/metallb/metallb/v0.9.5/manifests/metallb.yaml the toleration is specified on the manifest.
So I feel it is better to keep it as the default.

Reply:
Agreeed. Adressed in d47e38b.

Reply:
Thanks for updating again.
Please rebase for the latest master branch to pass CI tests.
I will add `/lgtm` again after rebasing also.

/lgtm